operator/buffer/memory.go

```diff
@@ -212,35 +213,36 @@ func (m *MemoryBuffer) newClearer(ids []uint64) Clearer {
 func (m *MemoryBuffer) Close() error {
 	m.inFlightMux.Lock()
 	defer m.inFlightMux.Unlock()
-	return m.db.Update(func(tx *bbolt.Tx) error {
-		memBufBucket, err := tx.CreateBucketIfNotExists([]byte("memory_buffer"))
-		if err != nil {
-			return err
-		}
-
-		b, err := memBufBucket.CreateBucketIfNotExists([]byte(m.pluginID))
-		if err != nil {
-			return err
-		}
-
-		for k, v := range m.inFlight {
-			if err := putKeyValue(b, k, v); err != nil {
-				return err
-			}
-		}
-
-		for {
-			select {
-			case e := <-m.buf:
-				m.entryID++
-				if err := putKeyValue(b, m.entryID, e); err != nil {
-					return err
-				}
-			default:
-				return nil
-			}
-		}
-	})
+	// return m.db.Update(func(tx *bbolt.Tx) error {
+	// 	memBufBucket, err := tx.CreateBucketIfNotExists([]byte("memory_buffer"))
+	// 	if err != nil {
+	// 		return err
+	// 	}
+
+	// 	b, err := memBufBucket.CreateBucketIfNotExists([]byte(m.pluginID))
+	// 	if err != nil {
+	// 		return err
+	// 	}
+
+	// 	for k, v := range m.inFlight {
+	// 		if err := putKeyValue(b, k, v); err != nil {
+	// 			return err
+	// 		}
+	// 	}
+
+	// 	for {
+	// 		select {
+	// 		case e := <-m.buf:
+	// 			m.entryID++
+	// 			if err := putKeyValue(b, m.entryID, e); err != nil {
+	// 				return err
+	// 			}
+	// 		default:
+	// 			return nil
+	// 		}
+	// 	}
+	// })
```

Comment:
Is this temporary? If so, what's missing? If not, we should delete it.

Reply:
Same as above